src/test/regress/regress_gp.c

```diff
@@ -628,6 +631,39 @@ repeatPalloc(PG_FUNCTION_ARGS)
 	PG_RETURN_INT32(0);
 }
 
+/*
+ * This function is a copy of resGroupPalloc that existed before the patch.
```

Comment:
What patch? :D
Can you be a little more specific for future readers?

Reply:
Done